proprietary/assets/package.json

```diff
@@ -26,6 +26,6 @@
     "url": "git@github.com:nl-design-system/rvo.git"
   },
   "devDependencies": {
-    "sass": "^1.69.7"
+    "sass": "^1.71.1"
```

Comment:
Vreemd dat dit `^x.y.z` dakje toegestaan was, hebben we in deze repo geen package-json-lint?

Reply:
Geen idee waarom deze er tussendoor geglipt is maar die hele folder is foetsie nu dus. Ik resolve deze.